flexxamples/howtos/openlayers.py

```diff
@@ -0,0 +1,177 @@
+from flexx import flx
+import os
+
+BASE_DIR = os.getcwd()
+
+with open(BASE_DIR + '/static/js/ol-v4.6.4/ol.css') as f:
+    ol_css = f.read()
+
+with open(BASE_DIR + '/static/js/ol-v4.6.4/ol.js') as f:
+    ol_js = f.read()
+
+flx.assets.associate_asset(__name__, 'ol.css', ol_css)
+flx.assets.associate_asset(__name__, 'ol.js', ol_js)
```

Comment:
Can this not be loaded from a CDN instead?

Reply:
Yes it can. But with this example I also wanted to show how you can load local assets so your app can work offline and also get feedback if this is the right way to do that? But if you prefer to use a CDN I'll change it no problem.

Reply:
In general I prefer using CDN, because it keeps the Flexx repo small and clean. But I agree that showing the "local variant" also has value, so let's leave it like this :)

Reply:
Could you add a docsting to this example that briefly explains what this howto demonstrates, including how it loads local assets?